docs/standard/library-guidance/dependencies-net.md

```diff
@@ -0,0 +1,169 @@
+---
+ai-usage: ai-assisted
+---
+# Guidance for .NET library dependency versions
+
+When building libraries that target multiple .NET versions, choosing dependency versions has implications for compatibility, servicing, and ecosystem health. This document outlines the main strategies, their tradeoffs, and recommendations.
+
+---
+
+## **Overview**
+
+Library authors face challenges when deciding which version of a .NET dependency to reference. Newer versions have more API and features, but may require a local redistribution increasing servicing responsibilities of the library and size of the application.  The decision impacts:
+
+- **Friction of updates** on older runtimes
+- **Engineering complexity** in maintaing the solution
+- **Servicing complexity** in managing supported releases
+
+This guidance provides options, tradeoffs, and a decision matrix to help you choose the best approach.
+✔️ DO be deliberate in choosing a dependency policy for your library
+✔️ DO remove out of support target frameworks from your package.
+✔️ CONSIDER choosing an approach that minimizes engineering costs and adjusting based on customer feedback
+✔️ CONSIDER changing your approach through the lifecycle of your library
+❌ DO NOT assume any policy is incorrect, all policies are technically sound and supported with different tradeoffs for consumption
+
+---
+
+## **Strategies for Dependency Version Selection**
+
+### **Option 1: Latest supported versions**
+
+Reference the latest supported version of the dependency across all target frameworks.  For example: reference 10.0 packages on `netstandard2.0`, `net8.0`, `net9.0`, and `net10.0`
```

Comment:
This should probably be qualified with a date this was written or a statement about what release is current; otherwise, it'll be out-of-date when .NET 11 ships.